src/phases/execution/ClusteredCollection.yml

```diff
@@ -6,8 +6,14 @@ Description: |
   key such as monotonically increasing inserts, range queries and range deletions.
 
 GlobalDefaults:
-  RecordIdExtraPaddingParam: &RecordIdExtraPaddingParam {^Parameter: {Name: "RecordIdExtraPadding", Default: 0}}
-  PaddingGenerator: &PaddingGenerator {^FastRandomString: {length: *RecordIdExtraPaddingParam, alphabet: "a"}}
+  RecordIdExtraPaddingParam:
```

Comment:
I'm assuming any changes after this are lint / whitespace only?

Reply:
The [commits](https://github.com/mongodb/genny/pull/1205/commits) tab lists all the commits. Only the following commits are not lint/whitespace related. That said, I totally understand that the mixture of manual fixes and different formatters has made the PR hard to review. Please allow me sometime to re-org the changes to make it hopefully easier to review.
- [DEVRPOD-5390 update .yamllint](https://github.com/mongodb/genny/pull/1205/commits/5689626ff6475b39d872503e8e44a3babc063cb2)
- [turn on yaml linting for evergreen check](https://github.com/mongodb/genny/pull/1205/commits/5c5c8b0673b7b6d4532c88e8e15a1e45a5f99ec7)
- [Merge branch 'master' into lungang.fang/DEVPROD-5390_lint-genny-take2](https://github.com/mongodb/genny/pull/1205/commits/fa26a8cf84a4f8cf72336deb10185eeb0a043413)

Reply:
Thanks ! that helps alot. I was a bit confused by the merge commit but then I was also on the other PR review.